machines/Issuers/IssuersActions.ts

```diff
@@ -55,6 +63,7 @@ export const IssuersActions = (model: any) => {
     }),
     setSelectedCredentialType: model.assign({
       selectedCredentialType: (_: any, event: any) => event.credType,
+      wellknownKeyTypes: (_: any, event: any)=> event.credType.proof_types_supported.jwt.proof_signing_alg_values_supported as string[]
```

Comment:
proof_types_supported will always have jwt as key?

Reply:
Good point 🤔

Reply:
Add condition for jwt
If proof supported type is jwt, get values from proof_signing_alg_values_supported
else fallback to old RSA key